src/v/resource_mgmt/tests/CMakeLists.txt

```diff
@@ -6,3 +6,22 @@ rp_test(
   LIBRARIES v::seastar_testing_main v::resource_mgmt v::config
   LABELS resource_mgmt
 )
+
+# NB: Some of these rely on global state (low watermark of available_memory) so need to run in a separate binary
```

Comment:
Hmm, do you have an example of how this fails when mixed in a binary with other tests?

These are per-shard values, and a test runs exclusively on a shard from start to end, so I'd have though there wasn't interference (of course the value at test start might be unpredictable, but usually you are doing "delta" tests) so it doesn't matter?

Is the problem that earlier tests may have pushed the LWM down pretty low so then this test cannot necessarily push it lower?

Reply:
Current approach seems fine, thanks for the explanation.